dvl-a50/mavlink2resthelper.py

```diff
@@ -271,7 +333,7 @@ def send_vision_speed_estimate(self, speed_estimates):
 
         post(MAVLINK2REST_URL + '/mavlink', data=data)
 
-    def send_vision_position_estimate(self, timestamp, position_estimates, attitude_estimates=[0.0, 0.0, 0.0]):
+    def send_vision_position_estimate(self, timestamp, position_estimates, attitude_estimates=[0.0, 0.0, 0.0], reset_counter=0):
         "Sends message VISION_POSITION_ESTIMATE to flight controller"
```

Comment:
It seems redundant to comment this here, and (unnecessarily) ties it to the MAVLink message in use. Probably this function should be renamed to `send_position_estimate`, and the template should be renamed to `position_estimate_template` (which are their respective purposes within the code), and then the fact that it's being sent via MAVLink's `GLOBAL_VISION_POSITION_ESTIMATE` message is isolated to where the template is defined. Same reasoning applies to the other functions (e.g. `send_vision` -> `send_position_delta`).

If we're leaving it with the current naming and comment, the comment should at least be correct :-)
```suggestion
        "Sends message GLOBAL_VISION_POSITION_ESTIMATE to flight controller"
```

Reply:
I updated the relevant places to prepend `global_`. I understand it is a bit redundant, but I prefer to be explicit and redundant, for now...